CHANGELOG.md

```diff
@@ -35,6 +35,12 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+## [v0.37.13] - 2020-05-29
```

Comment:
You don't know the release date yet.

```suggestion
## [v0.37.13] - TBD
```

Reply:
well i would like to do the release today, if this gets approved and merge then i can create the release branch asap and then tag

Reply:
Yes, right, but you still add the release date on `release/...` before you push and tag -- not on the RC branches.